bandwidth-file-spec.txt

```diff
@@ -972,6 +972,9 @@ software_version=1.0.4
 
 A.5 Generated by sbws version 1.1.0
 
+Bandwidth Lines are wrapped here with new lines for readability,
+but they are not wrapped in the Bandwidth Files.
```

Comment:
We can't wrap lines. If we do, people will write parsers which parse the wrapped lines.
(Test vectors need to be exactly the same as the real implementation.)